### ISSUE TITLE
No license file

### ISSUE THREAD

**Issue by danry25** [blackc2004/fusionpbx-clicktocall#1]
Any chance you plan to add a license to this repo? As is, most people won't touch code that doesn't have a license. Hard to know what the author intended and all that jazz.

**Comment by blackc2004** [comment]
Hmm. Let me look into that... I hadn't even thought of that.

**Comment by danry25** [comment]
Yeah, each license has its benefits and downsides, I'm a fan of AGPLv3 (Notably used for MongoDB, Nextcloud and Owncloud), but FusionPBX itself is licensed under MPL. I license my extensions to FusionPBX under AGPLv3 to encourage contributions back, as while I want to encourage people to enhance the apps I wrote, if they start selling it as part of a hosted service to customers with modifications to the apps I wrote, AGPLv3 requires contributing their modifications back.
